### DISCUSSION TITLE
希望添加更多熱鍵

### DISCUSSION THREAD

**Discussion by alctny** [he3-app/he3-feedback#62] [Ideas]
希望添加如下熱鍵：
Ctrl + d：將當前功能添加到收藏
Ctrl + w：關閉程序窗口，退到後台運行
Ctrl + q：完全退出程序
Ctrl + ]：收起/展開右側側邊欄（關聯工具）
Ctrl + [：收起/展開左側側邊欄（功能列表）

P.S: 另外希望修改默認喚起鍵，Alt+Space 在 Windows 和部分 Linux 桌面環境中爲窗口操作快捷鍵（順便一說：Ctrl+Space，Windows+Space 同樣容易熱鍵衝突）

P.S.S: 既然 Ctrl + k 輸入 ip 可直接獲取本機 IP 信息，輸入 Time 可以獲取當前時間信息，希望黨輸入數學表達式的時候能夠直接進行計算

**Comment by hongchaodeng** [comment]
非常好的建议，衷心感谢你的反馈，谢谢🙏
